audio-format-service/requirements.txt

```diff
@@ -1,14 +1,26 @@
 # Audio Format Service Dependencies
-fastapi>=0.104.0
-uvicorn>=0.24.0
-pydantic>=2.0.0
-python-multipart>=0.0.6
-ffmpeg-python>=0.2.0
-mutagen>=1.47.0
-opentelemetry-api>=1.20.0
-opentelemetry-sdk>=1.20.0
-opentelemetry-exporter-otlp>=1.20.0
-opentelemetry-instrumentation-fastapi>=0.41b0
-
-# Local shared package
--e ../shared
\ No newline at end of file
+
+# Shared package (editable install)
+-e ../shared
+
+# Web framework
+fastapi==0.104.1
+uvicorn[standard]==0.24.0
+python-multipart==0.0.6
+
+# Audio processing
+ffmpeg-python==0.2.0
+mutagen==1.47.0
+
+# HTTP client for testing
+httpx==0.25.2
+
+# Observability
+opentelemetry-api==1.21.0
+opentelemetry-sdk==1.21.0
+opentelemetry-instrumentation-fastapi==0.42b0
+
+# Development and testing
+pytest==7.4.3
+pytest-asyncio==0.21.1
+pytest-cov==4.1.0
```

Comment:
check all packages in context7 MCP what we are using the recent versions, unless you have strong evidences why we need to pin to specific old verion